src/scss/C23_WMDE_Desktop_DE_06.scss

```diff
@@ -0,0 +1,194 @@
+@use '@src/scss/settings/units';
+@use 'sass:map';
+@use 'sass:color';
+
+$primary: #3366CC;
+$error: #DD3333;
+$white: #FFFFFF;
+$black: #000000;
+$gray-mid: #B7B7B7;
+
+$input: (
+	'border-radius': 8px,
+	'border': 1px solid $gray-mid,
+	'height': 42px,
+	'font-size': 14px,
+	'min-width': 272px,
+	'max-width': 384px,
```

Comment:
this does not seem to be set properly, in my browser the input fields render as 100% width 
![Bildschirmfoto vom 2023-09-28 11-42-49](https://github.com/wmde/fundraising-app-frontend/assets/11670701/dbbe9876-5c61-47a7-87ff-8ed41d9411e0)

Reply:
ah I guess we're kind of waiting for Gabriels PR to be finished here...